examples/shared_samples/switch.rb

```diff
@@ -109,8 +111,8 @@
 # A scope defines a collection of resources, which might be used for filtering or access control.
 # When a scope uri is added to a switch resource, this resource is grouped into a resource(enclosure, server hardware, etc.) pool.
 # Once grouped, with the scope it's possible to restrict an operation or action.
-# For the switch resource, this feature is only available for C7000 and api version higher than or equal to 300.
-if @client.api_version >= 300 && variant == 'C7000'
+# For the switch resource, this feature is only available for C7000 and api version equal to 300.
+if @client.api_version == 300 && variant == 'C7000'
```

Comment:
Hey @AnushaY1916 , as discussed yesterday, can you check this condition? Please test it against 3.10 DCS to check if endpoint does not work with 500 also.

Also, this condition is not in line with the endpoint support file.